code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -94,11 +94,13 @@
 	icon_state = "cmo_cypherkey"
 	channels = list("Medical" = 1, "Command" = 1)
 
+/*CITADEL CHANGE - comments this out to be overridden in modular_citadel
```

Comment:
You don't have to comment out the entire type here, just override what's changed in modular_citadel

Reply:
I commented the whole encryption key and headset out for the HoP since I was basically changing the entire item.